core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,28 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.generateDummyExplorations = function() {
+          // Generate dummy explorations with random title.
+          AdminTaskManagerService.startTask();
+          var POSSIBLE_TITLES = ['Hulk Neuroscience',
```

Comment:
Consider breaking after "[". That way you aren't pushing everything to the right.